doc/nrf/known_issues.rst

```diff
@@ -979,6 +979,47 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-13594: The channel map provided by the LE Host Set Channel Classification HCI command is not always applied on the secondary advertising channels.
+In this case, the extended advertiser can send secondary advertising packets on channels which are disabled by the Host.
+This can result in a scanner that is unable to receive some advertising packets.
```

Comment:
Sorry, missed this one in the first iteration. 
```suggestion
```
The scanner will likely be able to receive the packets anyways unless there is a lot of radio noise on some channel

Reply:
Shall I just write "may result" then? To leave it ambiguous?

Reply:
I would suggest to leave it out